docs/quickstarts/excel-quickstart-content.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Build your first Excel content add-in
+description: Learn how to build a simple Excel content add-in by using the Office JS API.
+ms.date: 06/20/2024
+ms.service: excel
+ms.localizationpriority: medium
+---
+
+# Build an Excel content add-in
+
+In this article, you'll walk through the process of building an Excel [content add-in](../design/content-add-ins.md) using Visual Studio.
+
+## Create the add-in
+
+### Prerequisites
+
+[!include[Quick Start prerequisites](../includes/quickstart-vs-prerequisites.md)]
+
+### Create the add-in project
+
+1. In Visual Studio, choose **Create a new project**.
+
+1. Using the search box, enter **add-in**. Choose **Excel Web Add-in**, then select **Next**.
+
+1. Name your project **ExcelWebAddIn1** and select **Create**.
+
+1. In the **Create Office Add-in** dialog window, choose **Insert content into Excel spreadsheets** add-in type, then choose **Next**.
```

Comment:
This is minor, but I think including the article here will help with localization 
```suggestion
1. In the **Create Office Add-in** dialog window, choose the **Insert content into Excel spreadsheets** add-in type, then choose **Next**.
```